Userland/Applications/DisplaySettings/MonitorSettingsWidget.cpp

```diff
@@ -100,6 +100,7 @@ ErrorOr<void> MonitorSettingsWidget::generate_resolution_strings()
 
     return {};
 }
+
```

Comment:
Oh, looks like you amended the wrong commit. I'd remove the newline again, run `git commit -a --amend` to restore this commit, then add the line back, do `git commit -am fixup`, then run `git rebase -i master`, move the fixup commit below the first commit, and mark it as `f`. That should work.

Reply:
Did as instructed, the newline now appears on my side of the repo at the very least. This time also pushed with --force-with-lease instead of just --force as per your comment on the previous review. Is it corrected now?